gensrc/proto/internal_service.proto

```diff
@@ -70,6 +70,7 @@ message PTransmitChunkParams {
 
     // Some statistics for the runing query
     optional PQueryStatistics query_statistics = 8;
+    optional bool use_pass_through = 9 [default = false] ;
```

Comment:
Why thrift default is false, session variables default value is true?

Reply:
Yes, we can set default value to true here.